environment.yml

```
@@ -0,0 +1,13 @@
+dependencies:
```

Many of these only exist in conda-forge so you should specify the channels:
```suggestion
channels:
  - conda-forge
  - defaults
dependencies:
```